mentoring/tests/integration/test_data_export.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
```

Comment:
@bradenmacdonald The will come back soon : ) Harvard asked to reimplement it in a more scalable way. We might be able to use the reporting framework you are specifying for this, but since they want a lot of live search (like an interface where you type the name of a student and see all his answers), I'd like to see how instantaneous the results can be before moving this data to XBlock fields.